### PR TITLE
Turn on merges into internal branches

### PR DESCRIPTION
**Pull request by wtgodbe**
Concurrently to merging this, we should disable public subscriptions and enable internal subscriptions (where did we land on 3.1.2xx vs 3.1.3xx for May?)

### CODE REVIEW

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -1554,66 +1554,66 @@
...
+        "https://github.com/dotnet/aspnetcore/blob/release/3.1/**/*",
         "https://github.com/dotnet/aspnetcore-tooling/blob/release/2/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.0/**/*",
+        "https://github.com/dotnet/aspnetcore-tooling/blob/release/3.1/**/*",
         "https://github.com/dotnet/blazor/blob/release/2/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/blazor/blob/release/3.1/**/*",
+        "https://github.com/dotnet/blazor/blob/release/3.0/**/*",
```
Comment (mmitche) [inline Maestro/subscriptions.json]: You can just remove all the 3.0 entries here.

Comment (wtgodbe) [reply Maestro/subscriptions.json]: Done

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -1554,66 +1554,47 @@
...
-        // "https://github.com/dotnet/Scaffolding/blob/release/3.1/**/*",
+        "https://github.com/dotnet/Scaffolding/blob/release/3.1/**/*",
         "https://github.com/aspnet/websdk/blob/release/2/**/*",
-        // "https://github.com/aspnet/websdk/blob/release/3.0/**/*",
-        // "https://github.com/aspnet/websdk/blob/release/3.1/**/*",
+        "https://github.com/aspnet/websdk/blob/release/3.1/**/*",
         "https://github.com/dotnet/cli/blob/release/2/**/*",
-        // "https://github.com/dotnet/cli/blob/release/3.0/**/*",
-        // "https://github.com/dotnet/cli/blob/release/3.1/**/*",
+        "https://github.com/dotnet/cli/blob/release/3.1/**/*",
```
Comment (wtgodbe) [inline Maestro/subscriptions.json]: @mmitche is this still the right format w/ the 3.1.1xx, 3.1.2xx, 3.1.3xx branches?

Comment (mmitche) [reply Maestro/subscriptions.json]: Yes. You can just imagine that meastro completetely removes `/**/*` and does a StartsWith match.

**Pull request merged**